### PR TITLE
Add testing

### PR DESCRIPTION
**Pull request by Chris-Schnaufer**
## Description

Added testing for betydb2json.py file

## Types of changes

This is the start of testing for this repository

## Checklist:

- [x] I have read the documentation on [CONTRIBUTING](https://github.com/AgPipeline/Organization-info) 
- [ ] My updates require a change to the documentation 
- [ ] I have updated the documentation accordingly
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed

## Time frame for completing Pull Request review (from creation date):
- [x] As soon as possible
- [ ] In the next two weeks
- [ ] In the next month

## Focus of Pull Request:

Reviewers should focus on the following aspects of this pull request. All comments are welcome regardless of which items are checked off.
- [ ] Inform others of new features and changes
- [x] There are no errors or inconsistencies
- [ ] Everything appears clear and easy to understand
- [x] Suggest improvements
- [ ] Identify areas where functionality and/or documentation is missing

## Hints to reviewers:

This builds off of the [remove_jx2json_app PR](https://github.com/AgPipeline/drone-makeflow/pull/6)
@julianpistorius No special instructions
@dlebauer No special instructions
@jvanderleeuw No special instructions

## Optional: issue links this Pull Request resolves

closes agpipeline/issues-and-projects#246

### CODE REVIEW

**Comment by Chris-Schnaufer** [comment]
Reopening and then repointing merge request to develop branch

File: .github/workflows/pylint_check.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      - name: Fetching pylint.rc file
+        run: wget https://raw.githubusercontent.com/AgPipeline/Organization-info/master/pylint.rc
+        id: fetch_pylint_resource
+      - name: Listing
+        run: ls -la
+      - name: Files to be linted
+        run: cat action_pylint_files.txt
+      - name: Running pylint
+        run: cat action_pylint_files.txt | xargs python3 -m pylint --rcfile ./pylint.rc
+      - name: run pytest
```
Comment (julianpistorius) [inline .github/workflows/pylint_check.yaml:74]: `pytest` isn't really part of linting. Tests should probably be in their own workflow, or change the workflow name to indicate it's more than just linting.

Comment (Chris-Schnaufer) [reply .github/workflows/pylint_check.yaml:74]: I agree. That's something left over from Ken's approach and the workflow name should change IMO. Thanks for catching that.

Comment (Chris-Schnaufer) [reply .github/workflows/pylint_check.yaml:74]: https://github.com/AgPipeline/issues-and-projects/issues/247

**Pull request merged**